ember-cli-build.js

```diff
@@ -2,13 +2,7 @@
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
 
 module.exports = function (defaults) {
-  var app = new EmberAddon(defaults, {
-    babel: {
-      optional: ['es7.decorators']
```

Comment:
Don't still need this?  Can't remember which package don't need this for.

Reply:
It is used by `ember-computed-decorators` which has been removed since it is not being used in this add-on.